dev_guide/templates.adoc

```diff
@@ -51,8 +51,16 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
+== Creating a Project From a Template
+
+You can create a project from a template that has been uploaded to your OpenShift instance. After a template has been uploaded, you can create a new project from either the Management Console or the CLI.
+
+On the Management Console, click on the Create button on the project page. It will then ask you to choose the template to create from, or upload a new template.
```

Comment:
@bparees we do not offer an 'upload' function unless this refers to a 'osc create -f <template>'.  I have considered adding some functionality to eclipse which would allow you to 'create app' from a template that you select from disk